docs/proposals/scheduler/README.md

```diff
@@ -0,0 +1,144 @@
+
+# Orka - Cluster Scheduler
+
+## Requirements
+
+### Service provided
+
+The Scheduler is responsible for the scheduling of nodes and workloads.
```

Comment:
I don't think the scheduler is responsible for scheduling nodes.

```suggestion
The Scheduler is responsible for scheduling workloads across the cluster nodes.
```

Reply:
[Done](https://github.com/dev-sys-do/orka/pull/2/commits/c05d47b3a14bf86d2dfef03cd4951d34367a2390#diff-a921ddd7f6a17dde09ca710e0254a45c8cf87c34b6872502dc8c80d5ab43c556R8)!